### ISSUE TITLE
TypeError in Craft 3 migration

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-forms#259]
```
TypeError: Return value of barrelstrength\sproutforms\migrations\m180417_000000_sproutforms_craft3_schema_changes::safeUp() must be of the type boolean, none returned

Migration: barrelstrength\sproutforms\migrations\m180417_000000_sproutforms_craft3_schema_changes

Output:

Exception: Return value of barrelstrength\sproutforms\migrations\m180417_000000_sproutforms_craft3_schema_changes::safeUp() must be of the type boolean, none returned (/home/tricopaint/craft3/vendor/barrelstrength/sprout-forms/src/migrations/m180417_000000_sproutforms_craft3_schema_changes.php:24)
#0 /home/tricopaint/craft3/vendor/craftcms/cms/src/db/Migration.php(56): barrelstrength\sproutforms\migrations\m180417_000000_sproutforms_craft3_schema_changes->safeUp()
#1 /home/tricopaint/craft3/vendor/craftcms/cms/src/db/MigrationManager.php(243): craft\db\Migration->up(true)
#2 /home/tricopaint/craft3/vendor/craftcms/cms/src/db/MigrationManager.php(163): craft\db\MigrationManager->migrateUp(Object(barrelstrength\sproutforms\migrations\m180417_000000_sproutforms_craft3_schema_changes))
#3 /home/tricopaint/craft3/vendor/craftcms/cms/src/services/Updates.php(225): craft\db\MigrationManager->up()
```

### Notify

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_i29f)

**Comment by boboldehampsink** [comment]
This is related to #260 and #255

**Comment by BenParizek** [comment]
This is resolved in v3.0.0-beta.49
